presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -48,23 +48,19 @@ public class PrestoNativeQueryRunnerUtils
     public static final String REMOTE_FUNCTION_JSON_SIGNATURES = "remote_function_server.json";
     public static final String REMOTE_FUNCTION_CATALOG_NAME = "remote";
 
+    public static final NativeWorkerParameters NATIVE_WORKER_PARAMETERS = getNativeWorkerParameters();
```

Comment:
Nit : getNativeWorkerParameters seems too generic. Maybe restrict it to getNativeWorkerPaths ?

Reply:
It also retrieves the worker count, so it's not just paths. Do you have another suggestion?

Reply:
Calling this NativeQueryRunnerParameters is better I think. NativeWorkerParameters is confusing given all the configs we have around.